app/Views/errors/html/error_exception.php

```diff
@@ -289,9 +291,7 @@
                                 continue;
                             }
 
-                            if (! is_array($value)) {
-                                $value = [$value];
-                            } ?>
+                            $value = [$value]; ?>
```

Comment:
this change due to `$headers` is from `headers()` method which got `array<string, Header>` docblock return, so it always a `Header` detected, so to loop, it wrapped with `[]`, but probably incorrect docblock, it needs to be verified.

@MGatner  @kenjis @paulbalandan could you verify ? Thank you.

Reply:
`$request->headers()` returns `array<string, Header>`.
https://github.com/codeigniter4/CodeIgniter4/blob/29c31173b6ecdd3be75ea69b3f5eb876e7b5dcbe/system/HTTP/MessageTrait.php#L116
https://github.com/codeigniter4/CodeIgniter4/blob/29c31173b6ecdd3be75ea69b3f5eb876e7b5dcbe/system/HTTP/MessageTrait.php#L154

So `$value` is never an array.
I sent a PR to refactor: #7171